app/questionnaire/path_finder.py

```diff
@@ -55,7 +52,7 @@ def build_path(self, blocks, this_location):
             if this_location.block_id in self.CLOSING_INTERSTITIAL_PATH:
                 return path
 
-            block_index = Navigator._block_index_for_location(blocks, this_location)
+            block_index = PathFinder._block_index_for_location(blocks, this_location)
```

Comment:
It should as I don't believe anything uses it other than PathFinder, honestly I think this test should be removed as it's checking a private method.